pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -271,11 +290,15 @@ func (m *Manager) processShutdownEvent() error {
 	case <-c:
 		break
 	case <-time.After(m.shutdownGracePeriodRequested):
-		klog.V(1).InfoS("Shutdown manager pod killing time out", "gracePeriod", m.shutdownGracePeriodRequested)
+		msg := "Shutdown manager pod killing time out"
```

Comment:
IMO, I don't think is very valuable to have an event for event for. The main reason this can happen is if for whatever reason `killPod` has taken more than the requested time (e.g. if CRI has not responded in time). This is more of an warning log and I don't think we have traditionally have events for those. So I would keep this one as a log.